index.html

```diff
@@ -6,14 +6,324 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-    <!-- Add a link to your custom CSS here -->
+    
+    <!-- Importing Google fonts  -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
+    
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&display=swap" rel="stylesheet">
+    
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
 
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    
+                        <!-- Adding a nav bar and a toggler -->
+<header>
+    <section class="fixed-top bg-light">
+        <aside class="d-none d-lg-flex justify-content-end mx-5 pt-3">
+            <a class="booking" href="#">Book an Appointment</a>
+        </aside> 
+        
+        <nav class="navbar navbar-expand-lg bg-light  flex-column-reverse flex-lg-row px-lg-5 g-0">
+            <a class="mynav-brand navbar-brand" href="#">Maria Hope, Ph.D - Psychologist</a>
+           
+            <div class="ms-auto pe-1">
+                
+                <button class="navbar-toggler navbar-light" type="button" data-toggle="collapse" data-target="#navbarMenu">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                
+                <div class="collapse navbar-collapse" id="navbarMenu">
+                    <ul class="mynav-lists navbar-nav ms-auto">
+                        <li class="nav-item"><a class="nav-link" href="./index.html">Home</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#">About</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#">Services</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#">Blog</a></li>
+                        <li class="nav-item"><a class="nav-link" href="./contact.html">Contact</a></li>
+                    </ul>
+                </div>
+            
+            </div>
+                
+        </nav>
+    
+    </section>
+</header>
+                       
+<main>                  <!-- Adding top part of the page (a picture and "Online Therapy for Emotional Health" part) -->
+    <article class="top-page mx-lg-5">
```

Comment:
I would try to avoid putting words that describe the layout in the name of the class - e.g. **top**, **middle**, **bottom**. Leave layout specifications for the code in your CSS blocks.

The reason being, imagine i'm Maria who you've built this website for and now decide I want to change the order of the sections on the page. Not only will you need to rearrange the sections in `index.html` but you'll need to change all your class names too because they are all inaccurate now.

I'd suggest giving your sections names that correspond to the type on content in them. The items in the navigation bar give a good indication of what each of the sections is about; about, services, blog.

Reply:
Thank you for this suggestion.